modules/videoio/src/cap_dshow.cpp

```diff
@@ -2869,7 +2884,9 @@ int videoInput::start(int deviceID, videoDevice *VD){
     mt.majortype     = MEDIATYPE_Video;
 
     // Disable format conversion if using 8/16-bit data (e-Con systems)
-    if (checkSingleByteFormat(VD->pAmMediaType->subtype) || (VD->pAmMediaType->subtype == MEDIASUBTYPE_Y16)) {
+    if (checkSingleByteFormat(VD->pAmMediaType->subtype) ||
+        (VD->pAmMediaType->subtype == MEDIASUBTYPE_Y16 || VD->pAmMediaType->subtype == MEDIASUBTYPE_YUY2))
+    {
```

Comment:
Converting the YUY2 to BGR manually instead of using DirectShow helps with making the OBS Virtual Camera work. But I am not sure whether it is a bug in the OBS Virtual Camera or in the capture implementation.